pkg/actuators/machine/vm.go

```diff
@@ -142,7 +176,7 @@ func findVMByName(ntnxclient *nutanixClientV3.Client, vmName string) (*nutanixCl
 		return nil, err
 	}
 	if len(res.Entities) == 0 {
-		err = fmt.Errorf("Not Found VM by name %s. error: %v", vmName, err)
+		err = fmt.Errorf("Not Found VM by name %s. error: %s", vmName, "VM_NOT_FOUND")
```

Comment:
why not unwrap the error here?

also, i noticed that `%q` is being used in other log lines for the vm name, we should follow the same convention here.

Reply:
Changed as suggested.